src/commands/deployImage/imageSource/buildImageInAzure/RunStep.ts

```diff
@@ -38,6 +42,22 @@ export class RunStep extends AzureWizardExecuteStep<IBuildImageInAzureContext> {
     }
 
     public shouldExecute(context: IBuildImageInAzureContext): boolean {
-        return !context.run
+        return !context.run;
+    }
+
+    protected initSuccessOutput(): ExecuteActivityOutput {
+        // Skip here, success will be output by the build image step
+        return {};
+    }
+
+    protected initFailOutput(context: IBuildImageInAzureContext): ExecuteActivityOutput {
+        return {
+            item: new GenericTreeItem(undefined, {
```

Comment:
The failure won't be output by the build image step also?

Reply:
This one is kind of confusing - my understanding is basically the building starts in this `RunStep`... but the actual meaningful output gets detected in the following step `BuildImageStep`.  Thus, what we want to actually display on success doesn't exist yet, so I defer showing a success output until we can confirm it in the following step.

For the fail, we can still trigger a fail-point within this step, so we should still define the potential fail output here.